tools/auth/get_id_token.go

```diff
@@ -0,0 +1,64 @@
+package auth
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+// Key for verify custom token must be saved in PROJECTAPIKEY environment variable
+// example: PROJECTAPIKEY=<projectApiKey> go run get_id_token.go <customToken>
+
+// getIDToken verify customToken and return idToken
+func getIDToken(customToken string, projectAPIKey string) interface{} {
+	requestBody, err := json.Marshal(map[string]string{
+		"token":             customToken,
+		"returnSecureToken": "true",
+	})
+
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	resp, err := http.Post(
+		"https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyCustomToken?key="+projectAPIKey,
+		"application/json",
+		bytes.NewBuffer(requestBody))
+
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	var result map[string]interface{}
+	err = json.NewDecoder(resp.Body).Decode(&result)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	idToken := result["idToken"]
+	if idToken == nil {
+		log.Fatalln("Invalid projectAPIKey or customToken.")
+	}
+
+	return result["idToken"]
+}
+
+func printIDToken(customToken string, projectAPIKey string) {
+	fmt.Println(getIDToken(customToken, projectAPIKey))
+}
+
+func main() {
+	numberOfArguments := len(os.Args)
+
+	if numberOfArguments != 2 {
+		log.Fatalln("Arguments mismatch error. Provide only the custom token as a single argument.")
+	}
+
+	customToken := os.Args[1]
```

Comment:
might more sense to use flag or cobra libraries to handle the cli flags/args, it is more error-prone and more readable imho
simple flag parsing here https://github.com/google/exposure-notifications-server/blob/main/tools/exposure-client/main.go